setup-build-env

```diff
@@ -0,0 +1,6 @@
+wget https://github.com/gohugoio/hugo/releases/download/v0.80.0/hugo_0.80.0_Linux-64bit.tar.gz
```

Comment:
These need shebangs!

Reply:
Also, best to use either wget or curl throughout; some environments don't have one or the other, best to keep deps minimal, etc; I'd just replace lines 1-3 with `curl https://github.com/gohugoio/hugo/releases/download/v0.80.0/hugo_0.80.0_Linux-64bit.tar.gz | tar xv hugo`

Reply:
alsox2, use `set -e`